cmd/cli/kubectl-kyverno/utils/common/kyverno_policies_types.go

```diff
@@ -204,3 +205,66 @@ OuterLoop:
 
 	return engineResponses, nil
 }
+
+func combineRuleResponses(imageResponse engineapi.EngineResponse) engineapi.EngineResponse {
+	if imageResponse.PolicyResponse.RulesAppliedCount() == 0 {
+		return imageResponse
+	}
+	ruleResponses := imageResponse.PolicyResponse.Rules
+	var failRuleResponses []engineapi.RuleResponse
+	var errorRuleResponses []engineapi.RuleResponse
+	var passRuleResponses []engineapi.RuleResponse
+	var skipRuleResponses []engineapi.RuleResponse
+	ruleName := ruleResponses[0].Name()
+	ruleType := ruleResponses[0].RuleType()
```

Comment:
Rule name and type are fixed?

Reply:
yes it is fixed.

Reply:
Why? There could be multiple rules returned in the response, we'll lose information if fixing the name and type.